### ISSUE TITLE
Accessibility and Challenge

### ISSUE THREAD

**Issue by adnzzzzZ** [adnzzzzZ/blog#46]
Lately there's been a lot of argument over accessibility and challenge in games because of [Sekiro's release](https://store.steampowered.com/app/814380/Sekiro_Shadows_Die_Twice/). Namely, some people think the game should have an easy mode, others don't. So I thought I'd write down my thoughts on this.

<br>

### Personality

One of the things I think about when thinking through these issues is that often times what makes people enjoy certain types of games comes from their biological needs via their personalities. And these can be roughly separated with the [Big Five](https://en.wikipedia.org/wiki/Big_Five_personality_traits) and mapped into different game types like this ([source](https://quanticfoundry.com/)): 

<p align="center">
<img src="https://i.imgur.com/UbA6Msz.png"></a>
</p>

I don't think this mapping of the big five traits into gaming is final or 100% correct, but it's a good starting point to work from from a group of people who have done the work to get here. You can watch the explanation for how they did it here:

<p align="center">
<a href="https://www.youtube.com/watch?v=YZwiQd-0xqQ"><img src="https://i.imgur.com/ClkzLTW.png"></a>
</p>

For instance, I'm someone who doesn't really like challenging games. If a game has other aspects I enjoy and it just happens to be challenging I'll likely play it, but if it's challenging for the sake of challenge I likely won't play it. Because of this I don't really play any of the souls games and didn't play Sekiro. I really really like what the "design" and "power" aspects though, which deal generally with character customization and getting more and more powerful over time. So games like [Risk of Rain 2](https://store.steampowered.com/app/632360/Risk_of_Rain_2/) are very good for me. You get to try to lots and lots of different builds because there are so many items and over time your character gets more powerful.

The point is, these personality differences between people are real, they're biological, and people will be driven to play different types of games because of it. Which brings me to...

<br>

### Challenge

As you can see, challenge is one of the aspects identified by the Quantic Foundry people. Often what happens with AAA games is that developers will try to appeal to the biggest number of personality types as possible because they want more people buying their games. I think everyone understands this intuitively even without knowing about personality traits. But sometimes comes along a company like From Software and they decide that they'll focus on a personality trait that's largely ignored as it means a portion of the population isn't being addressed properly. And so they make the souls franchise which is largely focused on the challenge aspect.

One of the reasons the challenge aspect gets ignored by most big studios is because unlike other aspects, if you really want to cater to the types of people who enjoy challenge you have to exclude everyone who doesn't. This is by definition how the trait works and it's also how it works in real life. Many fields are zero sum, meaning if you win someone else has to lose, and people who like challenge are often driven to those fields. Politics is a good example of a zero sum field, given that there's a fixed number of positions available. 

In the case of single player games the competition with other people is indirect rather than direct: you're playing a single player game (so you're not competing directly), but beating the game at all is challenging and anyone who manages it has gone through the same amount of pain and challenge. This is what makes the game interesting and what makes those types of people play it. Adding an easy mode completely devalues the experience for the people who are in it for the challenge (and those people are the main audience of the game), because suddenly it means that beating the game doesn't mean much anymore.

Note that there have been big arguments before in the gaming community when it comes to other traits. I think the Gone Home debate was largely an argument of people who like the story and artistic aspects of games with people who largely don't. And this brings me to...

<br>

### Journalists/indie devs vs gamers

This is something that happens constantly and it's pretty interesting to watch, but journalists and certain indie devs seem to be always be "against" gamers and vice-versa. This challenge debate is another example of that, the Gone Home debate was also another example. In my view the reason this happens is because both of those groups are just fundamentally different personality wise. To be an indie developer or a journalist you have to be pretty high in the trait called [openness to experience](https://en.wikipedia.org/wiki/Openness_to_experience), which deals with creativity, interest for ideas and the artistic aspect of things. Whereas to be a gamer you just have to be a human being in the general population, and the general population won't be as high in openness as a group that is professionally selected for that trait.

So what happens is that, being high in openness, journalists/indie devs will enjoy certain aspects of games more than others. They'll enjoy the artistic appeal of games, they'll enjoy the story, they'll enjoy fantasy elements, but they generally won't enjoy things like grinding or challenge, since to enjoy those you have to be higher in [conscientiousness](https://en.wikipedia.org/wiki/Conscientiousness), and as a group journalists or indie devs aren't particularly selected for that (and I'd guess that because they're selected for openness often times what happens is that people high in openness will be lower in conscientiousness and vice-versa).

And so this creates a very natural conflict between the interests journalists and some indie devs and the interests of a big portion of the gaming population. And these interest differences will manifest themselves in things like asking for an easy mode to a challenging game. And one of the reasons why these debates tend to get so heated is because they're not just debates about games, they're about people's identity. If you're someone who enjoys challenge a lot that will shape the way you view the world and the way you act in it, it's not just a preference, it literally informs everything around you and how you parse reality. And so when other people attack that you will feel attacked as well. The same argument applies to people who really enjoy the story/artistic element of games. If you're someone like that and you see games that are about that getting attacked you'll also feel extremely defensive. It's a natural reaction and it's a part of being human.

<br>

### Possible solutions

The main solution to these problems is to simply understand that people have different preferences and that no one is right or wrong. Understand your own preferences and how they differ from other people's and frame your disagreements in this manner. I don't like games like Sekiro because I don't like challenge that much, but I understand how other people might enjoy it and how adding an easy mode would ruin the experience for them, so I won't ask for something like that. Similarly, I don't like games like Gone Home or The Witness, but I can understand why people would enjoy them, so I won't really ask for them to have better gameplay that I would enjoy more like powerups or something. It doesn't make any sense.

Finally, the main argument MANY people who support an easy mode make is, "what difference does it make if there's an easy mode?" Two examples of this (there are many many more):

<p align="center">
<img src="https://i.imgur.com/BPAloCa.png"></a>
</p>

<p align="center">
<img src="https://i.imgur.com/QZ429ms.png"></a>
</p>

And as I just explained it, the challenge aspect of the game would be ruined for the people who enjoy that aspect and who are largely ignored by most AAA developers. The reason why so many people don't seem to understand this is because they're not the target audience for FromSoft's games and so they don't feel that gut feeling of WRONG whenever someone mentions that the game should be made easier. Often times people also can't explain this gut feeling, it just feels wrong and they know they're against it with no real way to rationally explain it.

So I'd advise for anyone who gets into these arguments and who wants more accessibility in games to be more empathetic. FromSoft is catering to a very small group of people who really enjoys challenge and those people aren't catered to in a real way by anyone else (how many well made games from AAA studios are there with no easy mode?). By coming in and trying to get their experience ruined you're both attacking a minority of people and you're trying to force your own personality preferences on them. This isn't the correct way to get things done and I think that more empathy will help. 

People can have different preferences and games can cater to specific types of people at the exclusion of others. Not every game should cater to everyone because some personality traits by definition require exclusion, and challenge happens to be one of them.
